src/structures/Battlemetrics.js

```diff
@@ -801,6 +809,18 @@ class Battlemetrics {
         return ordered.map(e => e[1]);
     }
 
+    getUpcomingWipesOrderedByTime() {
```

Comment:
Does this run even if there is a server that does not give any details about upcoming wipes? Or is rust_wipes always available in the battlemetrics API call?

Reply:
Also add a comment header for the function